.travis.yml

```diff
@@ -42,7 +44,7 @@ install: composer update --prefer-source --no-interaction --optimize-autoloader
 
 script:
     - bin/phpunit --debug $( if [ $TEST_COVERAGE = true ]; then echo "-d xdebug.max_nesting_level=1000 --coverage-clover=build/logs/clover.xml"; fi; )
-    - if [ ${TRAVIS_PHP_VERSION} == "7.0" ]; then bin/php-cs-fixer fix --diff --dry-run -v; fi;
+    - if [ ${TRAVIS_PHP_VERSION} == "7.0" ]; then composer require --dev 'friendsofphp/php-cs-fixer:^2.0' && bin/php-cs-fixer fix --diff --dry-run -v; fi;
```

Comment:
Good idea! Let's make it run on 7.1 instead for speed?

Reply:
remove it from requirements because it seems to not be compatible with Symfony 4